client/components/QuestionsAnswers/AnswersList.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import { months } from './lib/dataFunctions.js';
+import api from '../../api.js';
+import { MarkAnswerHelpfulAndReported } from './MarkHelpfulAndReported.jsx';
+
+const AnswersList = ({answers, answersShow, showMoreA}) => {
+
+  return (
+    <Container>
+      <AnswerContainer>
+        <TitleA>A:</TitleA>
+        <AnswerContent>
+          {answers.sort((a, b) => { return (b.helpfulness - a.helpfulness); }).slice(0, answersShow).map((answer, i) => {
+            let timeArr = answer.date.split('T')[0].split('-');
+            return (
+              <div key={i}>
+                <div>{answer.body}</div>
+                <div>
+                  {answer.photos.map((photo, i) => {
+                    return (
+                      <Photos key={i}>
+                        <img src={photo} alt='photo'/>
+                      </Photos>
+                    );
+                  })}
+                </div>
+                <ByUser>
+                  <span>by { answer.answerer_name }, { months[timeArr[1]] } { timeArr[2] }, { timeArr[0] }&emsp;|&emsp;</span>
+                  {/* <span>Helpful?{answer.helpfuless} <u >Yes</u>({answer.helpfulness})&emsp;|&emsp;</span> */}
+                  <MarkAnswerHelpfulAndReported answer={answer}/>
+                </ByUser>
+              </div>
+            );
+          })}
+          {answersShow < answers.length && <LoadMA onClick={ showMoreA }>See more answers</LoadMA>}
+        </AnswerContent>
+      </AnswerContainer>
+    </Container>
+  );
+};
+
+const Photos = styled.div`
```

Comment:
In all of your styled components I'd make sure to explicitly set colors e.g. ${props => props.theme.textLight}